crates/nu-command/src/network/url/build_query.rs

```diff
@@ -14,7 +14,10 @@ impl Command for SubCommand {
         Signature::build("url build-query")
             .input_output_types(vec![
                 (Type::record(), Type::String),
-                (Type::table(), Type::String),
+                (
+                    Type::Table([("key".into(), Type::Any), ("value".into(), Type::Any)].into()),
```

Comment:
I don't understand why we only allow tables with key and value but no other tables? I think that's what this means.

Reply:
Previously this command treated one row tables as if they were records, and these two would be equivalent.
```nushell
({param_1: "aaa", param_2: "bbb"} | url build-query)
([[param_1, param_2]; ["aaa", "bbb"]]  | url build-query)
"param_1=aaa&param_2=bbb"
```
But multiple row tables resulted in unexpected behaviour
```nushell
[[param_1, param_2]; ["aaa", "bbb"], ["ccc", "ddd"]] | url build-query
"param_1=aaa&param_2=bbbparam_1=ccc&param_2=ddd"
```
where the results of two rows would be concatenated without a separator.

I wanted to add a structured way to provide key-pair values where duplicates would be allowed and the order would be preserved.
`table<key, value>` seemed the way to go about it, but continuing to accept one row tables would complicate both behavior and implementation.

Reply:
ok. i'm not necessarily against it. i was just wanting to understand better. it may be good to have some "extra_description()" to explain this, not sure.